pyserini/demo/msmarco.py

```diff
@@ -34,19 +40,68 @@ def precmd(self, line):
     def do_help(self, arg):
         print(f'/help    : returns this message')
         print(f'/k [NUM] : sets k (number of hits to return) to [NUM]')
+        print(
+            f'/model [MODEL] : sets encoder to use the model [MODEL] (one of tct, ance)')
+        print(
+            f'/mode [MODE] : sets retriver type to [MODE] (one of sparse, dense, hybrid)')
 
     def do_k(self, arg):
         print(f'setting k = {int(arg)}')
         self.k = int(arg)
 
+    def do_mode(self, arg):
+        if arg == "sparse":
+            self.searcher = self.ssearcher
+        elif arg == "dense":
+            if self.dsearcher is None:
+                print(f'specify model through /model before using dense retrieval')
```

Comment:
Msg should be complete sentences, e.g., 'Specify model through /model before using dense retrieval.'
Same for others.